app/javascript/shared/components/Spinner.jsx

```diff
@@ -0,0 +1,18 @@
+import { h } from 'preact';
+
+export const Spinner = () => (
+  <svg
+    className="crayons-spinner"
```

Comment:
Should we add `crayons-icon` class there as well? Since it could be potentially used elsewhere as replacement for other icons so maybe it could use our default `crayons-icon` styling. Orrrrr, apply that styling to `crayons-spinner` instead?

Reply:
Done